docs/usage.md

```diff
@@ -56,7 +56,7 @@ sample2,run1,ILLUMINA,sample2.unmapped_1.fastq.gz,sample2.unmapped_2.fastq.gz,sa
 The typical command for running the pipeline is as follows:
 
 ```bash
-nextflow run genomic-medicine-sweden/meta-val --input ./samplesheet.csv --outdir ./results -profile docker
+nextflow run genomic-medicine-sweden/meta-val --input ./samplesheet.csv --outdir ./results -profile docker --perform_extract_reads --extract_kraken2_reads
```

Comment:
I would suggest to have an example of how to run each workflow here.

Reply:
Done!